packages/ui/src/utils/componentDefinitions.ts

```diff
@@ -286,6 +286,20 @@ export const componentDefinitions = {
       inputWrapper: 'bui-InputWrapper',
       input: 'bui-Input',
       inputIcon: 'bui-InputIcon',
+      inputAction: 'bui-InputAction',
+    },
+    dataAttributes: {
+      invalid: [true, false] as const,
+      disabled: [true, false] as const,
+    },
+  },
+  PasswordField: {
+    classNames: {
+      root: 'bui-PasswordField',
+      inputWrapper: 'bui-InputWrapper',
+      input: 'bui-Input',
+      inputIcon: 'bui-InputIcon',
+      inputAction: 'bui-InputAction',
```

Comment:
These class names should be unique. You don't need to repeat them. The best way would be to use the class names from `TextField` directly. We do the same for `SearchField` I believe.